pages/favorites.js

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import styled from "styled-components";
+import Link from "next/link";
+import AnswerCard from "../components/AnswerCard";
+import { TfiHome } from "react-icons/tfi";
+
+export default function FavoriteAnswers({
+  morningAnswers,
+  eveningAnswers,
+  toggleFavorite,
+}) {
+  const stringToLocaleDate = (dateString) =>
+    new Date(dateString).toLocaleDateString("de-DE", {
+      weekday: "short",
+      day: "numeric",
+      month: "numeric",
+      year: "numeric",
+    });
+
+  return (
+    <>
+      <NavHeader>
+        <Link href="/">
+          <NavIconToHomepage />
+        </Link>
+      </NavHeader>
+      <Heading>Deine Highlights.</Heading>
+
+      <div>
+        {morningAnswers?.map((answerText) => {
+          if (answerText.isFavorite === true) {
+            return (
+              <div>
+                <StyledEntryDate>
+                  {stringToLocaleDate(answerText.createdDate)}
+                </StyledEntryDate>
+                <AnswerCard
+                  answerText={[answerText]}
+                  toggleFavorite={toggleFavorite}
+                />
+              </div>
+            );
+          } else {
+            return null;
+          }
+        })}
```

Comment:
**I would like to show you two other ways for conditional rendering, the first is my favorite 🙂** 

1. [Inline If with Logical && Operator](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator)
 ```suggestion
      {morningAnswers?.map(
          (answerText) =>
            answerText.isFavorite && (
              <div>
                <StyledEntryDate>{stringToLocaleDate(answerText.createdDate)}. 
               </StyledEntryDate>
                <AnswerCard answerText={[answerText]} toggleFavorite={toggleFavorite} />
              </div>
            )
        )}
```